docs/releases/5.1.txt

```diff
@@ -207,6 +207,11 @@ Models
   :class:`~django.db.models.TextField`, or
   :class:`~django.contrib.postgres.fields.ArrayField` can now be :ref:`sliced
   <slicing-using-f>`.
+* :attr:`~django.db.models.UniqueConstraint.violation_error_code` and 
```

Comment:
As the 5.1 alpha has been released, please rebase these changes and move this note to 5.2 and update the version notes to 5.2 👍